src/client/components/pages/revision.js

```diff
@@ -105,9 +112,29 @@ class RevisionPage extends React.Component {
 		return title;
 	}
 
+	constructor(props) {
+		super(props);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+	handleSubmit(event) {
+		event.preventDefault();
+		const data = {
+			note: this.noteInput.getValue()
+		};
+		console.log(data);
+		request.post(`/revision/${this.props.revision.id}/note`)
+			.send(data).promise()
+			.then(() => {
+				location.reload();
```

Comment:
Is this `reload()` copied from other components? I'd rather we not have to leave the page and instead just update inline, but I'm guessing other components just reload, so I won't ask that this change here. (This is more a note to myself.)